File: samples/workinprogress/pi-calculator/app/controllers/Global.scala
import play.api._

import java.util.concurrent._

object MyGlobal extends GlobalSettings {
  
  override def onStart(app: Application) {
    Logger.info("Start Pi-Calc")
  }
  
  override def onStop(app: Application) {
    Logger.info("Stop Actor System")
    controllers.Application.system.shutdown
  }
  
}

File: documentation/manual/javaGuide/main/global/code/javaguide/global/startstop/Global.java
package javaguide.global.startstop;

//#global
import play.*;

public class Global extends GlobalSettings {

    public void onStart(Application app) {
        Logger.info("Application has started");
    }

    public void onStop(Application app) {
        Logger.info("Application shutdown...");
    }

}
//#global
